index.js

```diff
@@ -139,6 +139,45 @@ function heartbeat() {
     this.isAlive = true;
 }
 
+
+function processMessage(call, type){
+    const https = require('https')
```

Comment:
Is it as easy a changing this to const http = require('http') and find/replace https with http ?